### ISSUE TITLE
Node Change Parent fails during keyboard navigation

### ISSUE THREAD

**Issue by mikeyww** [giuspen/cherrytree#310]
In the "Node Change Parent" dialog, after typing a target node name and pressing ENTER, pressing ENTER again does not move the node to the target.  Instead, an error occurs, or a "Node Right" action is performed.

Mike

**Comment by m1k3s0** [comment]
@mikeyww are you still encountering this issue in the latest version of CT?

**Comment by mikeyww** [comment]
It worked in my latest test with v1.4.0. Thank you.
